CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+- feat: Convert values in dictioanry to string when sanitize #509
```

Comment:
Can we change this to?
```suggestion
- feat: Add sentry_sanitize for NSArray #509
```
For me converting values in a dictionary is not exactly what this PR adds.

Reply:
Sure